labs/SettingsControls/src/SettingsExpander/SettingsExpander.cs

```diff
@@ -30,21 +30,11 @@ private void RegisterAutomation()
         if (Header != null && Header.GetType() == typeof(string))
```

Comment:
```
        if (Header is string headerString && headerString != string.Empty)
```
Can simplify using the C# `is` syntax. See a few places around here that could benefit from this.